### DISCUSSION TITLE
How to change the Datepicker value outside of Data Table.

### DISCUSSION THREAD

**Discussion by shishamchudal** [Power-Components/livewire-powergrid#229] [Q&A]
Hey there,

I have a really quick question. Is it possible to change the value of the date picker without using the date picker?

In my project, there are two different calendars used, One is AD(default in date picker) and the other is Bikram Sambat (BS) calendar. All the date is in the BS calendar and the date filtering only works in the AD calendar.

To filter the dates I would like to use my own date picker outside the data table and pass the value to the date picker inside the data table. I couldn't figure out a way how to pass the data to faltpickr date picker. 

```PHP
protected function getListeners()
    {
        return [
            'pg:datePicker-' . $this->tableName   => 'datePikerChanged',
            'pg:editable-' . $this->tableName     => 'inputTextChanged',
            'pg:toggleable-' . $this->tableName   => 'inputTextChanged',
            'pg:multiSelect-' . $this->tableName  => 'multiSelectChanged',
            'pg:toggleColumn-' . $this->tableName => 'toggleColumn',
            'eventRefresh'                        => '$refresh',
        ];
    }
```

I have found these listeners on the documentation. Can I emit an event from the date picker outside the data table?

**Comment by luanfreitasdev** [comment]
Hello @shishamchudal , I didn't quite understand what you want to do and how to work with different calendars, you can override the listeners, but it will only take effect after the dates are selected.

If you find a direction, please don't be too excited to send a PR, it's always welcome!

Thanks
